Packs/XMatters/Integrations/xMatters/xMatters.yml

```
@@ -0,0 +1,202 @@
+  name: password
+  type: 4
+  required: true
+- display: URL of an HTTP trigger in a flow.
+  name: url
+  defaultvalue: https://acme.xmatters.com
+  type: 0
+  required: true
+  additionalinfo: This is for creating events in xMatters from Cortex XSOAR. Used
+    for the xm-trigger-workflow command.
+- display: Fetch Type
+  additionalinfo: In a future release the xMatters incidents will be supported. For now it's just events.
+  name: fetch_type
+  defaultvalue: events
+  type: 15
+  required: true
+  options:
+  - events
+- display: Fetch alerts with status (ACTIVE, SUSPENDED)
+  name: status
+  type: 15
+  required: false
+  options:
+  - ACTIVE
+  - SUSPENDED
+- display: Priority of events to fetch
+  name: priority
+  type: 16
+  required: false
+  options:
+  - LOW
+  - MEDIUM
+  - HIGH
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: Property Name
+  name: property_name
+  type: 0
+  required: false
+  additionalinfo: Property Name to look for in xMatters. Used for importing events.
+- display: Property Value
+  name: property_value
+  type: 0
+  required: false
+  additionalinfo: Property Value to look for in xMatters. Used for importing events.
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  defaultvalue: 7 days
+  name: first_fetch
+  required: false
+  type: 0
+- display: Maximum number of incidents to fetch
+  defaultvalue: '50'
+  name: max_fetch
+  required: false
+  type: 0
+script:
+  commands:
+  - name: xm-trigger-workflow
+    arguments:
+    - name: recipients
+      required: true
+      description: Recipients of the xMatters Message
+    - name: subject
+      description: Subject of the xMatters Message
+    - name: body
+      description: Body of the xMatters Message
+    - name: incident_id
+      description: Incident ID of Incident referenced
+    - name: close_task_id
+      description: Id of task to close in playbook. Requires an incident_id as the
+        investigation id.
+    outputs:
+    - contextPath: xMatters.Workflow.request_id
+      description: Request ID from xMatters
+      type: string
+    description: sends the event to xMatters
+  - name: xm-get-events
+    arguments:
+    - name: request_id
+      description: The UUID returned from triggering a workflow.
+    - name: status
+      auto: PREDEFINED
+      predefined:
+      - ACTIVE
+      - SUSPENDED
+      - TERMINATED
+      description: Status of the event.
+    - name: priority
+      auto: PREDEFINED
+      predefined:
+      - LOW
+      - MEDIUM
+      - HIGH
+      description: Priority of the event.
+    - name: from
+      description: A date in UTC format that represents the start of the time range
+        you want to search.
+    - name: to
+      description: A date in UTC format that represents the end of the time range
+        you want to search.
```

Please add an example of the time in the description.

```suggestion
        you want to search.(i.e. `2020-01-01T00:11:22Z`)
```